{{cookiecutter.project_slug}}/Dockerfile__conda

```diff
@@ -1,21 +1,19 @@
-ARG PYTHON_IMAGE_TAG=4.8.2
+ARG IMAGE_TAG=22.9.0-2
+
+FROM condaforge/mambaforge:${IMAGE_TAG}
+
+LABEL maintainer="{{ cookiecutter.company_name if cookiecutter.company_name else cookiecutter.full_name }}"
 
-FROM continuumio/miniconda3:${PYTHON_IMAGE_TAG}
-{%- if cookiecutter.company_name %}
-LABEL maintainer="{{ cookiecutter.company_name }}"
-{% else %}
-LABEL maintainer="{{ cookiecutter.full_name }}"
-{% endif -%}
 WORKDIR /{{cookiecutter.module_name}}
 COPY . .
```

Comment:
can we copy just `environment.yml` and `environment-dev.yml` first, install all dependencies, and then copy all the remaining stuff? this would make much more efficient use of docker layer caching, since we wouldn't have to rebuild the entire environment on each run

Reply:
Good point, I haven't looked at this. I updated the PR but no need to copy the `environment-dev.yml` imo as this is the application image.